tests/v2_validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -93,7 +93,7 @@
 RANCHER_DNS_SERVER = "169.254.169.250"
 RANCHER_DNS_SEARCH = "rancher.internal"
 RANCHER_FQDN = "rancher.internal"
-SERVICE_WAIT_TIMEOUT = 120
+SERVICE_WAIT_TIMEOUT = int(os.environ.get('SERVICE_WAIT_TIMEOUT', "120"))
```

Comment:
Just curious: why does this need to be overridable? Is there large variance in the necessary timeout depending on how we test?